shell/gpu/gpu_surface_gl_skia.cc

```diff
@@ -169,8 +170,7 @@ bool GPUSurfaceGLSkia::CreateOrUpdateSurfaces(const SkISize& size) {
   TRACE_EVENT0("flutter", "UpdateSurfacesSize");
 
   // Either way, we need to get rid of previous surface.
-  onscreen_surface_ = nullptr;
-  fbo_id_ = 0;
+  view_unit = nullptr;
```

Comment:
This seems really weird. We should evict it from the map here right?

Or we should just reset the surface on the pointer here?